src/System.Private.ServiceModel/tests/Benchmarks/PerfUsingCoreWCFService/CoreWCFPerfService/SayHello.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace WCFCorePerfService
+{
+    public class SayHello : ISayHello
+    {
+        public Task<string> HelloAsync(string name)
+        {
+            return Task.Factory.StartNew(() => { return name; });
+        }
```

Comment:
Task.FromResult(name) would be better to return as it will execute faster and we're testing the client here.